app/services/check_answers/sections/partner_income.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module CheckAnswers
+  module Sections
+    class PartnerIncome < Base
+      def initialize(check)
+        super(check:, label: :partner_income)
```

Comment:
Do we add a conditional, to only show the 'partner income' label when there is a partner? I get this additional heading when I remove the partner. 

![Screenshot 2024-08-12 at 13 42 31](https://github.com/user-attachments/assets/ce1139d6-35e5-473a-be97-27a6decdf1ab)